atom/browser/native_window.h

```diff
@@ -105,13 +105,13 @@ class NativeWindow : public base::SupportsUserData,
   virtual bool IsMinimized() = 0;
   virtual void SetFullScreen(bool fullscreen) = 0;
   virtual bool IsFullscreen() const = 0;
-  virtual void SetBounds(const gfx::Rect& bounds) = 0;
+  virtual void SetBounds(const gfx::Rect& bounds, bool animate) = 0;
```

Comment:
We should add default value for `animate` so we don't have to pass `false` in every call. Apparently only user would like to resize window with animation and all calls in Electron's code would just pass `false`.

```c++
void SetBounds(const gfx::Rect& bounds, bool animate = false);
void SetSize(const gfx::Size&, bool animate = false);
void SetPosition(const gfx::Point& position, bool animate = false);
void SetContentSize(const gfx::Size&, bool animate = false);
```